src/Model/Brand.php

```diff
@@ -48,8 +48,9 @@ public function getId(): int
 
     /**
      * @param int $id
+     * @return Brand
      */
-    public function setId(int $id)
+    public function setId(int $id): Brand
```

Comment:
non c'est faut.
Ici ton setId revoie :void
il faut que tu ajoutes éventuellement un return $this si tu veux qu'il renvoie un :Brand
mais ce n'est pas obligatoire